core/src/test/java/com/graphhopper/routing/util/parsers/AbstractBikeTagParserTester.java

```diff
@@ -100,6 +107,241 @@ protected double getSpeedFromFlags(ReaderWay way) {
         return avgSpeedEnc.getDecimal(false, edgeId, intAccess);
     }
 
+    @Test // Adapted from CarTagParserTest::testFerry, with motorcar and bicycle values swapped when given
```

Comment:
What do you mean with?
> swapped when given

Reply:
that this is a clone of `CarTagParserTest::testFerry` but where there was a `motorcar` tag value, I replaced it with `bicycle`

and the other way around also (so when `bicycle` value was used I put in `motorcar`)

so to make the test equivalent, but with Bikes as the focus

ref link to car test: https://github.com/graphhopper/graphhopper/blob/master/core/src/test/java/com/graphhopper/routing/util/parsers/CarTagParserTest.java#L478

Reply:
Ah ok, I thought so.